util/properties/properties.go

```diff
@@ -52,7 +52,7 @@ type SAnimationPropertyConfig struct {
 
 // bool values need to be parsed from 0 or 1
 
-type general struct {
```

Comment:
there is not really a need to capitalize those, my go-to (haha, get it?) format is camelCase, you can see more examples in hyprkeys.go

Reply:
dont we need to capitalize the first letter to have the struct be public

Reply:
Oh yeah that is true. Sorry, an oversight on my behalf.